### DISCUSSION TITLE
How does one create a transfer proposal?

### DISCUSSION THREAD

**Discussion by 0xQuasark** [near-daos/astro-ui#624] [Q&A] [answered]
How does one create a transfer proposal?

**Comment by 0xQuasark** [comment, marked as answer]
To create a transfer proposal you just need to fill description, amount and target (receiver account), link is optional. Then click on "Propose" and confirm transaction in near wallet

After your proposal gets approved, target account will receive amount you entered. But only if there is enough NEAR tokens on DAO balance.

Also keep in mind that 5 NEAR tokens are always locked in DAO, so balance must be 5 NEAR + amount you need to transfer.

**Comment by GaloisField2718** [comment]
Hello, 

How to transfer funds from a wallet to DAO on Astro. In this one https://testnet.app.astrodao.com/dao/anank.sputnikv2.testnet I can't add fund I have the error "The account anank.sputnikv2.testnet wouldn't have enough balance to cover storage, required to have 1679890340066263800000000 yoctoNEAR more."
I don't want to transfer from DAO to wallet but from wallet to DAO. How can I do that ?

**Reply by GaloisField2718** [reply]
I have created this proposal : https://explorer.testnet.near.org/transactions/GTgmCQYF7rU8osjEjUdZJi68cW6dCoiDs87J2EwXDD6q
